promscale/alerting.md

```diff
@@ -0,0 +1,80 @@
+# Alerting rules in Promscale
+Promscale supports alerting rules. It evaluates these rules
+at regular intervals. The Promscale connector can be used as the ruler while
```

Comment:
I don't understand what is meant by "ruler" in this context?

Reply:
Hi @Loquacity I guess this term was borrowed from https://thanos.io/tip/components/rule.md/#rule-aka-ruler. @VineethReddy02 Please let me know if my understanding is right?